dev/NavigationView/TestUI/NavigationViewRS3Page.xaml

```diff
@@ -214,7 +214,6 @@
                                     <VisualState x:Name="TitleBarVisible" />
                                     <VisualState x:Name="TitleBarCollapsed">
                                         <VisualState.Setters>
-                                            <Setter Target="PaneButtonGrid.Margin" Value="0,32,0,0"/>
```

Comment:
Backout this change. NavigationViewRS3Page.xaml means that we are trying to verify the scenario when customer have this ControlTemplate in RS3

Reply:
But the markup doesn't have the element in it -- was the style that we shipped in the OS in RS3 missing this element too?

Reply:
yes. we are missing it since RS3, but I don't know why it didn't crash

Reply:
Thanks for the details :) ! Change done ;)